### ISSUE TITLE
CSV conversion error

### ISSUE THREAD

**Issue by johnbwoodruff** [wakatime/wakadump#6]
When I run the following command:

``` shell
$ wakadump --input Wakatime.json --output csv
```

I get the following error:

``` shell
Traceback (most recent call last):
  File "/usr/local/bin/wakadump", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/wakadump/cli.py", line 49, in main
    formatter.run()
  File "/usr/local/lib/python2.7/site-packages/wakadump/formats/csv.py", line 26, in run
    self._get_column_names()
  File "/usr/local/lib/python2.7/site-packages/wakadump/formats/csv.py", line 51, in _get_column_names
    with click.progressbar(self.data['days'],
KeyError: 'days'
```

**Comment by alanhamlett** [comment]
Is your input file from a WakaTime Data Dump created on this page? https://wakatime.com/settings/account

**Comment by alanhamlett** [comment]
Fixed and released in v2.0.4.
